Models/INSERT.ts

```diff
@@ -0,0 +1,20 @@
+import { Request , Response } from "express";
+import { StudentProps } from "../Interfaces/studentinterface";
```

Comment:
studentModels. inside of it we have insert.ts update.ts select.ts

Reply:
from librarry we have this call==> studentModesl.select.function...

Reply:
inside index.ts we have 

import {studentSelect} from "./studentSelect.ts"

export const studentModels{
 select: studentSelect
update:studentupdate,
insert:studentInsert
}

Reply:
inside studentSelect.ts 

export const studentSelect{
}

export default studentSelect

Reply:
Done